package.json

```diff
@@ -17,13 +18,16 @@
   "author": "Ben Schmidt",
   "license": "ISC",
   "dependencies": {
-    "chalk": "^2.4.2",
-    "cli-cursor": "^3.1.0",
-    "figures": "^3.2.0",
+    "chalk": "^5.2.0",
+    "cli-cursor": "^4.0.0",
+    "figures": "^5.0.0",
     "lodash": "^4.17.21",
-    "rxjs": "^6.5.2"
+    "rxjs": "^7.8.0"
+  },
+  "peerDependencies": {
+    "inquirer": "^9.0.0"
```

Comment:
This communicates to the consumer what the supported inquirer versions are _and_ that the consumer must provide inquirer